packages/tpetra/core/src/Tpetra_CrsGraph_decl.hpp

```diff
@@ -1976,6 +2008,9 @@ namespace Tpetra {
     bool haveLocalConstants_;
     //! Whether all processes have computed global constants.
     bool haveGlobalConstants_;
+    //! Whether row and column maps match.
+    // Mutable for lazy evaluation.
```

Comment:
Will Doxygen pick up the second comment line, if it's just `//` instead of `///` etc.?

Reply:
No, and that is the goal. This is a implementation comment for developers, no reason for it to be in Doxygen.